src/config.c

```diff
@@ -417,6 +425,26 @@ int load_config(struct vpn_config *cfg, const char *filename)
 			cfg->user_cert = strdup(val);
 			if (strncmp(cfg->user_cert, "pkcs11:", 7) == 0)
 				cfg->use_engine = 1;
+		} else if (strcmp(key, "dns") == 0) {
+			free(cfg->domain_suffix);
+			cfg->domain_suffix = val;
+			continue;
+		} else if (strcmp(key, "dns") == 0) {
```

Comment:
AFAICS this block is never reached due to the same if-clause above.

Reply:
I guess the string being checked for in the above condition should have been like "domain-suffix", but not "dns".